## Third Year Committee Report
Sarah Stevens, McMahon Lab  
Meeting: July 8th, 2015 in MSB 5503

### Research Progress
Over the past year I have worked on tracking a set of thirty sequence-discrete populations from Trout Bog, as they are represented by genomes assembled from metagenomes(GFMs), using sixty-three metagenomic samples from six years. I called single nucleotide polymorphisms (SNPs) and looked for genes gained or lost by the populations.  One genome showed a genome-wide sweep of diversity through the time series.  Other genomes show evidence of past gene-sweeps, as reveled in regions of  significantly lower diversity.  From these results, we propose that diversity within different co-occuring populations may be predominantly controlled by different mechanisms(recombinaton vs. selection).  I currently have a co-first author manuscript submitted to ISMEJ on the results of this analysis.  

I am also currently working on a manuscript concerning the population dynamics of the ubiquitous and abundant freshwater Actinobacterial lineage called bacterium acI.  We have 14 single cell genomes(SAGs) from this lineage, representing four lakes and five 'tribes'.  We mapped reads from the Lake Mendota time series onto the five SAGs that were recovered from Lake Mendota in order to track changes within these sequence-discrete populations through time.  The results prompted us to examine how SAGs from the same and different lakes would map to one another.  We shredded the genomes to create a synthetic metagenome and mapped the shreds against each SAG.  We found that SAGs from different tribes do not map well SAGs from other tribes.  In fact, even SAGs from the same tribe do not always map well to each other.  When we mapped reads from Lake Mendota to the SAGs, we found that SAGs from collected from different lakes showed a different average nucleotide identity distribution among the hits.  We interpret this to mean that the population represented by those SAGs have diverged from the allopatric populations present in Lake Mendota.   <!--- Something about the gene patterns through time and coverage differences ?? -->  

#### New Questions
I continue to be interested in discerning the forces shaping wild bacterial populations, in order to understand how lineages diversify or remain cohesive. My work over the past year has generated new questions that address this research challenge.  

*Are there related sequence-discrete populations in TB and CB?  How closely related are they?  Do they share a common gene pool?* We now have GFM's from Crystal Bog (CB), which is of similar location and trophic status to Trout Bog (TB).  With these genomes, I will ask if there are related populations and how similar they are at the genome scale.  I will also investigate if these populations are really separated or if they share a common genepool.  I will compare the reference genomes and map reads from the opposite source.  From the acI SAGs we saw that reference genomes collected from different lakes do not recruit metagenomic reads with high ANI.

*Are all populations sequence-discrete?  What do non-seq-discrete groups look like? How does ANI structure of sequence-discrete populations compare between different lineages?* Most of the populations we have studied must be sequence-discrete in order to assemble into reference genomes from metagenomes.  One of my next goals is to investigate and compare population structures using references not assembled from communities.  Using both our SAGs and our metagenomes, as well as other marine and soil datasets of metagenomes paired with SAGs and sequenced fosmids, I will investigate population structure.  I am interested to see if I can find a population that does not fit the sequence-discrete pattern.  I have found some preliminary evidence that the LD12 and acTH1 lineages have different structures, in our freshwater datasets.  Along the same lines, I would like to quantify how the population structure, using ANI, is different for different lineages.

\newpage

### Publications
Bendall, M. L.\*, **Stevens, S. L. R.**\*, Chan, L.-K., Malfatti, S., Schwientek, P., Tremblay, J., … Malmstrom, R. R. (_submitted to ISMEJ_). Genome-wide selective sweeps and gene-specific sweeps in natural bacterial populations. ^\*Equal\ contributors^  
Garcia, S. L.\*, **Stevens, S. L. R.**\*, Crary, B., Martinez-Garcia, M., Oyserman, B., Stepanauskas, R., Woyke, T., Tringe, S. G., Andersson, S., Bertilsson, S., Malmstrom, R.,  McMahon, K. D. (_in prep for ISMEJ_). Genetic populations of abundant and uncultivated freshwater actinobacteria inferred from single cell genomes and metagenomes. ^\*Equal\ contributors^  
**Stevens, S. L. R.**, Hamilton, J. J., McMahon, K. D. (_in prep for SIGS_). Freshwater GFMs and SAGs from the Betaproteobacteria lineage betI.  
**Stevens, S. L. R.**, Hamilton, J. J., McMahon, K. D. (_in prep for SIGS_). Freshwater GFMs and SAGs from the Betaproteobacteria lineage betII.  
**Stevens, S. L. R.**, Hamilton, J. J., McMahon, K. D. (_in prep for SIGS_). Freshwater GFMs and SAGs from the Betaproteobacteria lineage betIV.  

### Oral Presentations
**Stevens, S. L. R.**, Bendall, M. L., Chan, L.-K., Malfatti, S., Schwientek, P., Tremblay, J., … Mcmahon, K. D. Malmstrom, R. R. Genome-wide Selective Sweeps in Natural Bacterial Populations Revealed by Time-series Metagenomics. 15th International Symposium on Microbial Ecology. August 24, 2014. Seoul, South Korea.  [https://goo.gl/6iunz0](https://goo.gl/6iunz0)  
**Stevens, S. L. R.**, Bendall, M. L., Chan, L.-K., Malfatti, S., Schwientek, P., Tremblay, J., … Mcmahon, K. D. Malmstrom, R. R. Genome-wide and Gene-specific Selective Sweeps in Freshwater Bacterial Populations Revealed Using Metagenomics. 14 Symposium Society for Aquatic Microbial Ecology. August 2015. Uppsala, Sweden

### Poster Presentations
**Stevens, S. L. R.**, Bendall, M. L., … Mcmahon, K. D. Malmstrom, R. R. Dynamics of Sequence-Discrete Bacterial Populations Inferred Using Metagenomes. 15th International Symposium on Microbial Ecology. August 24, 2014. Seoul, South Korea. [https://goo.gl/qsYL32](https://goo.gl/qsYL32)  
**Stevens, S. L. R.**, Bendall, M. L., … Mcmahon, K. D. Malmstrom, R. R. Dynamics of Sequence-Discrete Bacterial Populations Inferred Using Metagenomes. SciMed GRS Poster Session 2014. Sept. 03, 2014. Madison, WI  
**Stevens, S. L. R.**, Garcia, S. Stepanauskas, R. Bertilsson, S. Malmstrom, R. R. McMahon, K. D. Time-resolved Metagenomics Reveals Population Expansion and Contraction in Freshwater Bacteria. Gordon Research Conference on Microbial Population Biology. August 2015.  

### Professional Development
* Software Carpentry Instructor Training - May 2015
* Molecular Microbial Ecology and Evolution(MoMiEE) support group - Started Nov. 2014 - Co-chair
* MoMiEE Python Study Group - Started Dec. 2014 - Chair
* Data Carpentry Hack-a-thon - Mar. 2015

### Committees and Outreach
+ MDTP Recruiting Committee 2014-2015
+ MDTP Steering Committee 2015-2016
+ Illinois Mathematics and Science Academy - Intersession Instructor - Bioinformatics Basics, Microbial Evolution and Ecology - Jan. 2015
+ Nuestro Mundo Science Night Volunteer - Apr. 2015
